supporting_content/metrics/negotiatedunitprice.md

```diff
@@ -0,0 +1,64 @@
+# Column: NegotiatedUnitPrice
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                     | Column                   |
+|:----------|:-----------------------------|:-------------------------|
+| AWS       | CUR;                         | *TODO: ?*                |
```

Comment:
**lineItem/NetUnblendedCost** - The actual after-discount cost that you're paying for the line item. This column is included in your report only when your account has a discount in the applicable billing period

Reply:
discussed this with Yuriy and we agreed that direct mapping in the AWS CUR file is not available. the same applies to the corresponding cost metric